### PR TITLE
fossdash_gsoc2020_part2

### PR DESCRIPTION
**Pull request by darshank15**
## Description

New features in fossdash, Also some configuration changes.

### CODE REVIEW

File: src/lib/php/common-sysconfig.php [hunk 1]
```diff
@@ -402,9 +402,9 @@ function Populate_sysconfig()
 
   $variable = "fossologyInstanceUUID";
   $fossologyInstanceUUIDPrompt = _('Fosslogy instance UUID');
-  $fossologyInstanceUUIDDesc = _('Set the fossology instance UUID. default autogenerated value if empty.');
+  $fossologyInstanceUUIDDesc = _('e.g. Instance UUID formate = [a-zA-Z0-9_-]* <br>Set the fossology instance UUID. Used autogenerated value if empty.');
```
Comment (NicolasToussaint) [inline src/lib/php/common-sysconfig.php]: Teh wording is a bit strange ; why not:
> Set the fossology instance UUID, leave empty to use autogenerated value.

File: src/lib/php/common-sysconfig.php [hunk 2]
```diff
@@ -386,25 +386,39 @@ function Populate_sysconfig()
...
+  $instanceUUIDValid = "check_fossology_instance_UUID";
+  $fossologyInstanceUUIDDesc = _('e.g. Instance UUID formate = [a-zA-Z0-9_-]+ <br>Set the fossology instance UUID. Used autogenerated value if empty.');
   $valueArray[$variable] = array("'$variable'", "null", "'$fossologyInstanceUUIDPrompt'",
-    strval(CONFIG_TYPE_TEXT), "'FossDashAPI'", "2", "'$fossologyInstanceUUIDDesc'", "null", "null");
+    strval(CONFIG_TYPE_TEXT), "'FossDashAPI'", "3", "'$fossologyInstanceUUIDDesc'", "'$instanceUUIDValid'", "null");
+
+  $variable = "FossdashReportedCleaning";
+  $FossdashReportingCleaningPrompt = _('Fossdash reported files cleaning');
+  $FossdashReportingCleaningValid = "check_fossdash_cleaning";
+  $FossdashReportingCleaningDesc = _('Set (in days)  older fossdash reported files should cleanup. Disable cleanup if empty.');
```
Comment (NicolasToussaint) [inline src/lib/php/common-sysconfig.php:419]: Could we change to :
> Number of days after which successfully pushed metrics are cleaned up.
> Set to Zero to disable cleanup.

File: src/lib/php/common-sysconfig.php (hunk 1 above)
Comment (darshank15) [reply src/lib/php/common-sysconfig.php]: Sure, Will update it, Thank you for the suggestion :). Mine was messy even grammatically,

**Comment by NicolasToussaint** [comment]
Merged to `dev/fossdash-exporter`

**Pull request closed without merging**